### ISSUE TITLE
Can't reassign Fn key on MacBook running Windows

### ISSUE THREAD

**Issue by Bougakov** [randyrants/sharpkeys#281]
Hi. I am using Windows on Apple's Macbook. Unlike HPs or Lenovos, it doesn't have BIOS (and BIOS setting to swap Fn and Ctrl keys), so I was hoping 'SharpKeys' would rescure me.

Similar program on Mac called [Karabiner Elements](https://karabiner-elements.pqrs.org/) did work for me: I've assigned a pair of "Ctrl -> Fn" modifications and that was it.

Tried the same on Windows wIth SharpKeys, but "Type a key" dialog doesn't react at all when I press Fn. Any ideas on how to address that? Thanks in advance.

**Comment by cakiya** [comment]
this is apple's issue right? i've googled around and basically 
- on normal keyboards the fn key physically doesn't send anything to the OS but...
- you CAN actually remap fn to control on a macbook running macOS

this means the macbook keyboard's fn key actually DOES send signals to the OS (which is why macOS can remap it) but apparantly bootcamp drivers that make the macbook keyboard work for Windows just don't send the signal.

so basically, the apple employee who coded the macbook keyboard driver for windows just decided that the fn key shouldn't send anything to Windows **(even thought it could),** which is why sharpkeys and everything windows key remapping doesn't work for macbook running windows right?

**Comment by randyrants** [comment]
Maybe is the best I can offer, only because on Windows, the Fn key is rarely seen on its own as well. Depends on how the whoever made the PC set up the Fn key: Windows sort of recognizes it, or at least a unique scancode has popped over over the years, but there's never a guarantee of it.

That said, there are plenty of features that Apple leaves out of Bootcamp, which I believe is out of laziness or an active willingness to prevent "star" Apple features from showing up in Windows, even when Windows _can_ support them.  For examples of this, see the trackpad on Apple laptops: nearly the same exact functionality in the HW yet when you boot Windows on Apple hardware, you get tap, two finger tap, and maybe detection of the lower right corner, for right click.  Gestures, scrolling, 3 and 4 finger support?  None of it, at the driver level, soooooo, maybe.
